ocaml/quicktest/quicktest_args.ml

```diff
@@ -64,6 +68,15 @@ let parse () =
       )
     ; ("-skip-xapi", Arg.Set skip_xapi, "SKIP tests that require XAPI")
     ; ("--", Arg.Rest_all set_alcotest_args, "Supply alcotest arguments")
+    ; ( "-only-run"
+      , Arg.String (fun x -> only_run := Some x)
```

Comment:
I'm not a native speaker but would find `run_only` more paradigmatic. But feel free to ignore.

Reply:
Changing it to `run-only` (which, interestingly, is the way I've described it in the commit message.....)